packages/schematics/webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const path = require('path');
```

Comment:
NOTE: This is fine for now, but planning on consolidating/refactoring the entire repo to use webpack with a base config and allow for overrides per project.